apps/web/playwright/booking-pages.e2e.ts

```diff
@@ -295,7 +295,8 @@ testBothFutureAndLegacyRoutes.describe("pro user", () => {
     await pageTwo.goto(initialUrl);
     await pageTwo.waitForURL(initialUrl);
 
-    await pageTwo.waitForSelector('[data-testid="event-type-link"]');
+    await pageTwo.waitForLoadState("networkidle");
```

Comment:
I'm not a fan of using waitingForLoadState. This means that either the loading strategy is not optimal or we're not using proper selector.

Reply:
From [the docs](https://playwright.dev/docs/api/class-frame#frame-wait-for-load-state):
 
<img width="938" alt="image" src="https://github.com/user-attachments/assets/ce448294-3bfd-40eb-adaa-1fbc34013a48">

Reply:
It's true for the `networkidle` arguments, but I found `waitingForLoadState` useful in certain conditions in some cases. Let me recheck if we can avoid using it in favor of a perfect selector instead.